pkg/ipamd/datastore/data_store.go

```diff
@@ -151,23 +154,35 @@ type ENI struct {
 	DeviceNumber int
```

Comment:
The reason I added `isPDEnabled` was because to avoid passing PD enabled flags to functions such as  "AssignPodIPv4Address" but you are right, I can make it global at the datastore level.